code/go/internal/jsonschema/common_spec.go

```diff
@@ -2,7 +2,7 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-package validator
+package jsonschema
```

Comment:
What about naming the package something more generic as "schema" or "specschema" ? I see that it is used gojsonschema library, but the spec files for instance are defined using YAML.

Reply:
Yeah, I called it this way because it uses the jsonschema, but it is right that our way to use it is quite custom, I will rename it.

Reply:
Ok, I have also split everything related to "yamlschema" to the already existing `yamlschema` package.

Everything related to the load of specs from directories, that is custom of this project, has been moved to `specschema`. Schema loading for files can be customized now, so we could replace "yamlschema" with other thing while keeping the logic for directory specs.